Makefile

```diff
@@ -1,6 +1,6 @@
 # Minimal script to install the SQL creation scripts ready for postinst script.
 
-VERSION=1.14.0dev
```

Comment:
Do I need to do this? what purpose does the dev suffi serve?

Reply:
I'm not sure. It could be so that locally built packages can be easily distinguished from released packages, but I don't see why you'd ever want to mix them.